.github/workflows/pcw.yml

```diff
@@ -14,7 +14,6 @@ jobs:
         python-version: ['3.10']
     steps:
       - uses: actions/checkout@v3
-      - uses: browser-actions/setup-geckodriver@latest
```

Comment:
why ?

Reply:
> why ?

It's redundant because geckodriver it's already in the VM used by GitHub Actions, and we risk running into rate-limits when pulling this action like it happened to me today.  See here:

https://github.com/SUSE/pcw/actions/runs/5647942036/job/15299079594

> Run browser-actions/setup-geckodriver@latest
> with:
> Error: API rate limit exceeded for 20.57.79.241. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

Reply:
:+1: